src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -13,7 +14,7 @@ import {imageConst} from '../../../../components/Image';
 
 const HUMAN_ID_URL = 'https://www.human-internet.org';
 
-const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, textSvg}) => {
+const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, text}) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the prop name from `textSvg` to `text` makes sense as the text is no longer being rendered as an SVG but as a string. This change improves readability and clarity.

```diff
- const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, textSvg}) => {
+ const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide, text}) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->